evm_loader/program/src/instruction/mod.rs

```diff
@@ -190,6 +196,7 @@ pub enum EvmInstruction {
     ///  `[WRITE]` Treasury
     ///  `[WRITE]` Operator Balance
     ///  `[]` System program
+    ///  `[]` Sysvar account
```

Comment:
It may be better to create a separate set of instructions with sysvar accounts.
That way we still have an option to run the transaction in the old format without it.

Discuss this with @afalaleev

Reply:
Ye, I discussed it. 
I managed to remove sysvar account and get the necessary data about compute budget instructions via syscall, so it's done.